docs/index.md

```diff
@@ -0,0 +1,99 @@
+|-------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Credentials | CircleCI requires an [API token](https://www.circleci.com/doc/book/using/using-credentials/) for all requests.                                                                                                                                                                                 |
+| Permissions | API tokens have the same permissions as the user who creates them, and if the user permissions change, the API token permissions also change.                                                                                                                                               |
+| Radius      | Each connection represents a single CircleCI Installation.                                                                                                                                                                                                                                   |
+| Resolution  | 1. With configuration provided in connection in steampipe _**.spc**_ config file.<br />2. With circleci environment variables.<br />3. An circleci.yaml file in a .circleci folder in the current user's home directory _**(~/.circleci/circleci.yaml or %userprofile\.circleci\circleci.yaml)**_. |
+
+### Configuration
+
+Installing the latest circleci plugin will create a config file (~/.steampipe/config/circleci.spc) with a single connection named circleci:
+
+```hcl
+connection "circleci" {
+  plugin = "circleci"
+
+  # Get your API token from CircleCI https://circleci.com/docs/api-developers-guide/#add-an-api-token
+
+  # api_token = "1234ee38fc6943f6cb9537a564e9a6dac6ef1463"
+}
+```
+
+By default, all options are commented out in the default connection, thus Steampipe will resolve your credentials using the same order as mentioned in [Credentials](#credentials). This provides a quick way to get started with Steampipe, but you will probably want to customize your experience using configuration options for querying multiple organizations, configuring credentials from your circleci configuration files, [environment variables](#credentials-from-environment-variables), etc.
+
+
+## Get involved
+
+- Open source: https://github.com/turbot/steampipe-plugin-circleci
+- Community: [Slack Channel](https://steampipe.io/community/join)
+
+## Configuring CircleCI Credentials
+
+### Credentials from Environment Variables
+
+The CircleCI plugin will use the standard CircleCI environment variables to obtain credentials **only if other arguments (`api_token`) are not specified** in the connection:
+
+#### API Token
+
+```sh
+export CIRCLE_TOKEN=1234ee38fc6943f6cb9537a564e9a6dac6ef1463
+```
```

Comment:
Do we need this section?

Reply:
@misraved why not? this is telling the user how to use an environment variable as creds. We got this section on other plugins

Reply:
Apologies @luisffc for the incorrect feedback on the need for this section.

I think instead of adding this section at the end of the file, we should add it just before the `Get involved` section as used in https://github.com/turbot/steampipe-plugin-azure/blob/main/docs/index.md#credentials-from-environment-variables

Reply:
Cool, just did it